### PR TITLE
add featureYK 

### PR DESCRIPTION
**Pull request by Kami-naga**
something about featureYK

### CODE REVIEW

File: featureFromKaminaga [hunk 1]
```diff
@@ -0,0 +1,4 @@
+some code
+if(condition statement){do something}else{do other things};
```
Comment (forestdan) [inline featureFromKaminaga]: The format is not easy to read. I think you should refactor this line.

Comment (Kami-naga) [reply featureFromKaminaga]: OK. I got that. I will refactor it now.

**Comment by Kami-naga** [comment]
I just changed the format. Pls check again.

File: featureFromKaminaga (hunk 1 above)
Comment (forestdan) [reply featureFromKaminaga]: Did a good job!

**Comment by forestdan** [comment]
> I just changed the format. Pls check again.

Did a good job!

**Pull request merged**